examples/cli/cli.cpp

```diff
@@ -1066,8 +1067,45 @@ int main(int argc, char ** argv) {
     }
 
     for (int f = 0; f < (int) params.fname_inp.size(); ++f) {
-        const auto fname_inp = params.fname_inp[f];
-		const auto fname_out = f < (int) params.fname_out.size() && !params.fname_out[f].empty() ? params.fname_out[f] : params.fname_inp[f];
+        const auto & fname_inp = params.fname_inp[f];
+        class FnameOutFactory {
```

Comment:
Could you rename this class to follow the [naming conventions](https://github.com/ggml-org/llama.cpp/blob/master/CONTRIBUTING.md#naming-guidelines). In this case something like `fname_out_factory` perhaps.

Reply:
Renamed to follow the requested naming conventions. It was to `fname_out_factory` before changes to "not output this log message" further renamed it to `fout_factory`. Note that now the struct and variable are homonyms.

`build/bin/whisper-cli -m models/ggml-tiny.en.bin -f samples/jfk.wav -oj -of - -np` no longer outputs `output_json: saving output to '/dev/stdout'`.